### PR TITLE
Mgb 197 implement include tables get habitat by id get request

### PR DESCRIPTION
**Pull request by samirehsani19**

### CODE REVIEW

File: MammalAPI/Configuration/Mapper.cs [hunk 1]
```diff
@@ -9,20 +9,9 @@ public class Mapper : Profile
     {
         public Mapper()
         {
-            CreateMap<Mammal, MammalDTO>().ForMember(
-                dto => dto.Habitats,
-                opt => opt.MapFrom(x => x.MammalHabitats
-                    .Select(y => y.Habitat)
-                    .ToList()))
-                .ReverseMap();
-
```
Comment (singelmamma) [inline MammalAPI/Configuration/Mapper.cs:18]: Ta bort ena MammalDTO mappern och implementera HabitatDTO mappningen

File: MammalAPI/DTO/MammalHabitatDTO.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using MammalAPI.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
```
Comment (singelmamma) [inline MammalAPI/DTO/MammalHabitatDTO.cs:6]: Ta bort denna DTO

File: MammalAPI/Controllers/HabitatController.cs [hunk 3]
```diff
@@ -47,7 +47,7 @@ public async Task<ActionResult<HabitatDTO[]>> GetAllHabitats(bool includeMammal
 
         ///api/v1.0/habitat/1    To get one habitat by id
         [HttpGet("{id:int}")]
-        public async Task<IActionResult> GetHabitatById(int id, [FromQuery]bool includeMammal = false)
+        public async Task<ActionResult<HabitatDTO>> GetHabitatById(int id, [FromQuery]bool includeMammal=false)
```
Comment (Gerglamesh) [inline MammalAPI/Controllers/HabitatController.cs:50]: Why are the spaces around "=" removed, I gues it's a matter of taste though...

File: MammalAPI/Controllers/HabitatController.cs [hunk 4]
```diff
@@ -61,9 +61,9 @@ public async Task<IActionResult> GetHabitatById(int id, [FromQuery]bool includeM
             }
         }
 
-        // /api/v1.0/habitat/name=pacific ocean                To get habitat by name
+       // /api/v1.0/habitat/name=pacific ocean                To get habitat by name
         ///habitat/name=Pacific Ocean?includeMammal=true       To get habitat by name and include mammal   
-        [HttpGet("{name}")]
+        [HttpGet("name={name}")]
```
Comment (Gerglamesh) [inline MammalAPI/Controllers/HabitatController.cs:66]: The convention we agreed on was to not use the "name=" style. Just "{name} should suffice unless there's a conflict that requires a new specified route.

**Pull request merged**